### PR TITLE
Adds parameters to the config file so that we could modify some of the query parameters

### PR DESCRIPTION
**Pull request by Mironor**

### CODE REVIEW

File: server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala [hunk 1]
```diff
@@ -126,7 +124,7 @@ class AddressIndexRepository @Inject()(
             matchQuery(
               field = "lpi.paoStartNumber",
               value = token
-            )),
+            ).boost(queryParams.buildingNumberLpiBoost)),
           tokensMap.get(Tokens.locality).map(token =>
```
Comment (saniemi) [inline server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala]: Do we test Tokens.locality against PAF.DEPENDENT_LOCALITY? That should be included if not...

Comment (Mironor) [inline server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala]: Nope, not currently the case, do you want a configurable boost on it ?

**Comment by Mironor** [comment]
@RichardSmithONS we did set all the default parameters in the config.

File: server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala (hunk 1 above)
Comment (saniemi) [inline server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala]: I don't think we need a configurable boost on Locality at this point. We need to modify the query anyway after we have re-indexed.

**Pull request merged**